pom.xml

```diff
@@ -1549,6 +1550,12 @@
         <version>${cglib.version}</version>
       </dependency>
 
+      <dependency>
```

Comment:
What is the new dependency on commons-chain needed for?

Reply:
That's in `dependencyManagement`. There were previously two versions being brought in, this restricts to only the latest version.